lab-adan/test/server-test.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const superagent = require('superagent');
+const expect = require('expect');
+const server = require('../lib/server.js');
+const GameScore = require('../model/gamescore.js');
+const PORT = process.env.PORT || 3000;
+
+describe('GameScore Constructor', () => {
+  it('Should return a GameScore Object', () => {
+    let newGame = new GameScore(1, 'tetris', 9000);
```

Comment:
Check your GameScore constructor parameter order and compare it to how you're supplying the arguments here.